src/test/FuzzerImpl.cpp

```diff
@@ -1593,6 +1561,36 @@ TransactionFuzzer::initializeTrustLines(AbstractLedgerTxn& ltxOuter)
     ltx.commit();
 }
 
+void
+TransactionFuzzer::initializeClaimableBalances(AbstractLedgerTxn& ltxOuter)
+{
+    LedgerTxn ltx(ltxOuter);
+
+    xdr::xvector<Operation> ops;
+
+    for (auto const& param : claimableBalanceParameters)
+    {
+        Claimant claimant;
```

Comment:
This code is fine, but history-wise this is not a good thing to do. Reviewing this is a lot harder because I don't get a side-by-side diff. Moving code should always be in a separate commit from changing it if possible (I can't think of any reasons this isn't possible off hand, but I suppose there could be some). No need to change it here, but this is important to keep in mind for the future.